docs/README.md

```diff
@@ -1,105 +1,411 @@
-# Geolocation GraphQL Interface
+# Geolocation GraphQL interface
 
-This app exports a GraphQL schema for geolocation results for Place Components.
+>⚠️ **USE AT YOUR OWN RISK**
+> 
+> *This is an ongoing, unsupported, unfinished and undocumented project. We do not guarantee any results after installation.*
 
-## Usage
+The Geolocation GraphQL interface provides autosuggestions for address inputs and autocompletion for address forms.
 
-### Installation
+>ℹ️ The Geolocation GraphQL interface allows you to choose which geolocation provider (e.g., [Google Maps](https://www.google.com.br/maps/), [Algolia Places](https://community.algolia.com/places/), [VTEX Places](https://github.com/vtex-apps/places-graphql), etc.) suits better for your store.
 
-To use it in your app, run the `vtex add vtex.geolocation-graphql-interface` command.
+It's possible to use the Geolocation GraphQL interface to connect with any front-end component, adapting the queries according to the UI needs.
 
-You may then use it in your front end component queries, for example, by writing the file `suggestAddresses.graphql`:
+Check the following sections for more information on how to use the data fetched from the Geolocation GraphQL interface in your React components.
 
-```graphql
-query SuggestAddresses($searchTerm: String!) {
-  suggestAddresses(searchTerm: $searchTerm)
-    @context(provider: "vtex.geolocation-graphql-interface") {
-    description
-    mainText
-    mainTextMatchInterval {
-      offset
-      length
-    }
-    secondaryText
-    externalId
-  }
-}
-```
+## Usage examples
 
-> ℹ️️ The only difference from a standard query is the usage of a _directive_: `@context(provider: "vtex.geolocation-graphql-interface")`
+>ℹ️ The following examples are based on the [Location Search](https://github.com/vtex-apps/place-components/#locationfield-search) component used during the checkout process in the [Place Components](https://github.com/vtex-apps/place-components) app.
 
-To resolve this query, you must have installed in your workspace an app that implements the GraphQL interface. Currently, those are the available resolvers:
+### Using autosuggestions for address inputs
 
-- [vtex.google-geolocation-resolver](https://github.com/vtex-apps/google-geolocation-resolver)
+Take the following example: imagine you want to develop a component that suggests a list of addresses based on a search term.
 
-Having zero or more than one resolver installed will result in an error.
+![location2](https://user-images.githubusercontent.com/60782333/100448177-ad8c2980-3090-11eb-8f9e-d01cfc7eef8f.png)
```

Comment:
That makes the explanation so much more clear, nice! I just feel like it's too blurred. If you think that makes sense, maybe we could ask the designers to edit the version from the design document?